astropy/cosmology/funcs.py

```diff
@@ -120,59 +120,59 @@ def z_at_value(func, fval, zmin=1e-8, zmax=1000, ztol=1e-8, maxfun=500,
+        at z=0 (default 1e-8).
 
     zmax : float or array-like['dimensionless'] or quantity-like, optional
-       The upper search limit for ``z`` (default 1000).
+        The upper search limit for ``z`` (default 1000).
 
     ztol : float or array-like['dimensionless'], optional
-       The relative error in ``z`` acceptable for convergence.
+        The relative error in ``z`` acceptable for convergence.
 
     maxfun : int or array-like, optional
-       The maximum number of function evaluations allowed in the
-       optimization routine (default 500).
+        The maximum number of function evaluations allowed in the
+        optimization routine (default 500).
 
     method : str or callable, optional
-       Type of solver to pass to the minimizer. The built-in options provided
-       by :func:`~scipy.optimize.minimize_scalar` are 'Brent' (default),
-       'Golden' and 'Bounded' with names case insensitive - see documentation
-       there for details. It also accepts a custom solver by passing any
-       user-provided callable object that meets the requirements listed
-       therein under the Notes on "Custom minimizers" - or in more detail in
-       :doc:`scipy:tutorial/optimize` - although their use is currently
-       untested.
+        Type of solver to pass to the minimizer. The built-in options provided
+        by :func:`~scipy.optimize.minimize_scalar` are 'Brent' (default),
+        'Golden' and 'Bounded' with names case insensitive - see documentation
+        there for details. It also accepts a custom solver by passing any
+        user-provided callable object that meets the requirements listed
+        therein under the Notes on "Custom minimizers" - or in more detail in
+        :doc:`scipy:tutorial/optimize` - although their use is currently
+        untested.
 
-       .. versionadded:: 4.3
+        .. versionadded:: 4.3
 
     bracket : sequence or object array[sequence], optional
-        For methods 'Brent' and 'Golden', ``bracket`` defines the bracketing
-        interval and can either have three items (z1, z2, z3) so that
-        z1 < z2 < z3 and ``func(z2) < func (z1), func(z3)`` or two items z1
-        and z3 which are assumed to be a starting interval for a downhill
-        bracket search. For non-monotonic functions such as angular diameter
-        distance this may be used to start the search on the desired side of
-        the maximum, but see Examples below for usage notes.
+         For methods 'Brent' and 'Golden', ``bracket`` defines the bracketing
+         interval and can either have three items (z1, z2, z3) so that
+         z1 < z2 < z3 and ``func(z2) < func (z1), func(z3)`` or two items z1
+         and z3 which are assumed to be a starting interval for a downhill
+         bracket search. For non-monotonic functions such as angular diameter
+         distance this may be used to start the search on the desired side of
+         the maximum, but see Examples below for usage notes.
```

Comment:
This looks like it's now 5 spaces, not 4. I think this is a mistake!

Reply:
Reading https://numpydoc.readthedocs.io/en/latest/format.html, all indents are 4 spaces.

Reply:
Thanks for catching - should be fixed now.